static/css/plugins/profilemanager.css

```
@@ -0,0 +1,1022 @@
+    text-transform: uppercase;
+}
+
+.welcome > .form.form-manager > .welcome-catalog > .manager-layout > .manager-form-pane > .manager-edit-banner > .manager-edit-banner-label > .manager-edit-banner-target {
+    background-color: rgba(115, 82, 0, 0.12);
+    border-radius: 2px 2px 2px 2px;
+    -o-border-radius: 2px 2px 2px 2px;
+    -ms-border-radius: 2px 2px 2px 2px;
+    -moz-border-radius: 2px 2px 2px 2px;
+    -khtml-border-radius: 2px 2px 2px 2px;
+    -webkit-border-radius: 2px 2px 2px 2px;
+    font-family: "SFMono-Regular", "Menlo", "Consolas", monospace;
```

Stylelint (17.11.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)